libs/langgraph/tests/test_pregel.py

```diff
@@ -6416,3 +6417,61 @@ def test_tags_stream_mode_messages() -> None:
             },
         )
     ]
+
+
+def test_node_destinations() -> None:
+    class State(TypedDict):
+        foo: Annotated[str, operator.add]
+
+    def node_a(state: State):
+        value = state["foo"]
+        if value == "a":
+            goto = "node_b"
+        else:
+            goto = "node_c"
+
+        return Command(
+            update={"foo": value},
+            goto=goto,
+            graph=Command.PARENT,
+        )
+
+    subgraph = StateGraph(State).add_node(node_a).add_edge(START, "node_a").compile()
+
+    def node_b(state: State):
```

Comment:
unrelated to this PR -- should we not have mypy fail this code due to missing type annotations?

Reply:
we're not running mypy on tests